kivy/uix/behaviors/focus.py

```diff
@@ -489,6 +487,22 @@ def _get_focus_next(self, focus_dir):
             else:
                 return None
 
+    def get_focus_next(self):
+        '''This method provides the next focuses(ie self.focus_next)
```

Comment:
There's a general style of the kivy docs that we should try to stick to. Mainly, it should be more succinct and use internal linking. There are also some grammar issues. E.g. a better version would be

```
Returns the next focusable widget using either :attr:`focus_next` or the :attr:`children` similar to the order when tabbing forwards with the ``tab`` key.
```